README.adoc

```diff
@@ -13,21 +13,20 @@ endif::[]
 
 == About
 
-Hawkular Metrics is the metric data store for the http://www.hawkular.org/[Hawkular] project.
-It can also be used independently.
+Hawkular Metrics is the metric data store engine part of http://www.hawkular.org/[Hawkular] community.
```

Comment:
`store` or `storage` ? Or do you want to separate the fact that Cassandra is the "real storage" ?